lib/terminology_table.erb

```diff
@@ -33,7 +33,10 @@
             <% binding['terminology'].each do |terminology| %>
               <% unless terminology['system'].nil? && terminology['display'].nil? %>
                   <li>
-                    <% if terminology['display'] && terminology['system'] %>
+                    <% #TODO: Adjust displays to be consistent once all resourced differentiate system from info url %>
+                    <% if terminology['display'] && terminology['system'] && terminology['info_link'] %>
```

Comment:
I *can* but I'm going to change what it means once I get through all the resources :)